listenbrainz/db/dump_manager.py

```diff
@@ -69,38 +69,32 @@ def create(location, threads):
 
 
 @cli.command()
-@click.option('--location', '-l', default=os.path.join(os.getcwd(), 'listenbrainz-export'))
+@click.option('--private-archive', '-pr', default=None)
+@click.option('--public-archive', '-pu', default=None)
+@click.option('--listen-archive', '-l', default=None)
 @click.option('--threads', '-t', type=int)
-def import_db(location, threads=None):
-    """ Import a ListenBrainz PostgreSQL dump into the PostgreSQL database.
+def import_dump(private_archive, public_archive, listen_archive, threads=None):
+    """ Import a ListenBrainz dump into the database.
 
-        Note: This method tries to import the private dump first, followed by the statistics
+        Note: This method tries to import the private db dump first, followed by the public db
             dump. However, in absence of a private dump, it imports sanitized versions of the
-            user table in the statistics dump in order to satisfy foreign key constraints.
-
-        Args:
-            location (str): path to the directory which contains the private and the stats dump
-            threads (int): the number of threads to use during decompression, defaults to 1
-    """
-    db.init_db_connection(config.SQLALCHEMY_DATABASE_URI)
-    db_dump.import_postgres_dump(location, threads)
+            user table in the public dump in order to satisfy foreign key constraints.
 
-
-@cli.command()
-@click.option('--location', '-l')
-@click.option('--threads', '-t', type=int)
-def import_listens(location=None, threads=None):
-    """ Import a ListenBrainz listen dump into the Influx database.
+        Then it imports the listen dump.
 
         Args:
-            location (str): path to the listenbrainz listen .tar.xz archive
-            threads (int): the number of threads to use while decompressing, defaults to 1
+            private_archive (str): the path to the ListenBrainz private dump to be imported
+            public_archive (str): the path to the ListenBrainz public dump to be imported
+            listen_archive (str): the path to the ListenBrainz listen dump archive to be imported
+            threads (int): the number of threads to use during decompression, defaults to 1
     """
-
-    if not location:
-        print('No location given!')
+    if not private_archive and not public_archive and not listen_archive:
```

Comment:
I think this is too simplistic. It doesn't make sense to import public and private at the same time. Should we allow importing listens without users/stats? I can't think of a compelling reason why not....

Reply:
Opened [LB-289](https://tickets.metabrainz.org/browse/LB-289) for now.